index.html

```diff
@@ -43,6 +43,9 @@ <h3 class='user-task'>Deep Breathing</h3>
             <button type="button" name="button" class='start-time'>START</button>
             <button type="button" name="button" class='log-activity hidden'>LOG ACTIVITY</button>
           </div>
+          <div class="completed-activity hidden">
```

Comment:
Looks, good!  Thanks for doing this, now we can begin to tackle the last bit of functionality!